### PR TITLE
chore(demo): update chip input demo

### PR DESCRIPTION
**Pull request by amcdnl**
This PR resolves issues with overflow due to out-dated example code. Ref: #7509

### CODE REVIEW

File: src/material-examples/chips-input/chips-input-example.html [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-  </mat-chip-list>
-  <input matInput placeholder="New fruit..."
+    <input matInput placeholder="New fruit..."
          [matChipInputFor]="chipList"
          [matChipInputSeparatorKeyCodes]="separatorKeysCodes"
          [matChipInputAddOnBlur]="addOnBlur"
          (matChipInputTokenEnd)="add($event)" />
-</mat-form-field>
+  </mat-chip-list>
+</mat-form-field>
```
Comment (mmalerba) [inline src/material-examples/chips-input/chips-input-example.html]: nit: \n

File: src/material-examples/chips-input/chips-input-example.html [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-  <mat-chip-list matPrefix #chipList>
+  <mat-chip-list #chipList>
     <mat-chip *ngFor="let fruit of fruits" [selectable]="selectable"
              [removable]="removable" (remove)="remove(fruit)">
       {{fruit.name}}
       <mat-icon matChipRemove *ngIf="removable">cancel</mat-icon>
     </mat-chip>
-  </mat-chip-list>
-  <input matInput placeholder="New fruit..."
+    <input matInput placeholder="New fruit..."
```
Comment (mmalerba) [inline src/material-examples/chips-input/chips-input-example.html]: remove `matInput`

Comment (amcdnl) [reply src/material-examples/chips-input/chips-input-example.html]: done

File: src/material-examples/chips-input/chips-input-example.html (hunk 1 above)
Comment (amcdnl) [reply src/material-examples/chips-input/chips-input-example.html]: done

**Pull request merged**